code/modules/surgery/organs/robolimbs.dm

```diff
@@ -3,142 +3,219 @@ GLOBAL_LIST_EMPTY(chargen_robolimbs)
 GLOBAL_LIST_EMPTY(selectable_robolimbs)
 GLOBAL_DATUM(basic_robolimb, /datum/robolimb)
 
+
+#define model        0
+#define brand        1
+#define childless    2
 /datum/robolimb
-	var/company = "Unbranded"                            // Shown when selecting the limb.
-	var/desc = "A generic unbranded robotic prosthesis." // Seen when examining a limb.
-	var/icon = 'icons/mob/human_races/robotic.dmi'       // Icon base to draw from.
-	var/unavailable_at_chargen                           // If set, not available at chargen.
-	var/selectable = 1									 // If set, is it available for selection on attack_self with a robo limb?
-	var/is_monitor										 // If set, limb is a monitor and should be getting monitor styles.
-	var/has_subtypes = 2								 // If null, object is a model. If 1, object is a brand (that serves as the default model) with child models. If 2, object is a brand that has no child models and thus also serves as the model..
+	var/company = "Unbranded"								// Shown when selecting the limb. Dio Unbrando.
+	var/desc = "A generic unbranded robotic prosthesis."	// Seen when examining a limb.
+	var/icon = 'icons/mob/human_races/robotic.dmi'			// Icon base to draw from.
+	var/unavailable_at_chargen = FALSE						// If TRUE, not available at chargen.
+	var/selectable = TRUE	// If set, is it available for selection on attack_self with a robo limb?
+							// This also affects character set up. Setting everything to TRUE by default. If we want to change this, let me know (Iren / Kiyahitayika)
+	var/is_monitor			// If set, limb is a monitor and should be getting monitor styles.
+	// If 0, object is a model. If 1, object is a brand (that serves as the default model) with child models. If 2, object is a brand that has no child models and thus also serves as the model..
+	var/has_subtypes = childless
 	var/parts = list("chest", "groin", "head", "r_arm", "r_hand", "r_leg", "r_foot", "l_leg", "l_foot", "l_arm", "l_hand")	// Defines what parts said brand can replace on a body.
 
+/* Arksoft's Tableau Entertainment */
+/datum/robolimb/tableau
+	company = "Tableau Entertainment Powered Mechanics"
+	desc = "This limb is made from a cheap polymer."
+	icon = 'icons/mob/human_races/cyberlimbs/nanotrasen/nanotrasen_main.dmi'
+	has_subtypes = childless
+	is_monitor = TRUE
+
+/* Bishop */
+//Main
 /datum/robolimb/bishop
 	company = "Bishop Cybernetics"
 	desc = "This limb has a white polymer casing with blue holo-displays."
 	icon = 'icons/mob/human_races/cyberlimbs/bishop/bishop_main.dmi'
-	has_subtypes = 1
+	has_subtypes = brand
 
-/datum/robolimb/bishop/alt1
-	company = "Bishop Cybernetics alt."
+/datum/robolimb/bishop/monitor
+	company = "Bishop Knight"
 	icon = 'icons/mob/human_races/cyberlimbs/bishop/bishop_alt1.dmi'
 	parts = list("head")
-	selectable = 0
```

Comment:
I defaulted everything to `selectable = TRUE`. If we want to change this back, sure, but I'm not sure how to make them available otherwise.